config/management/genesis/src/validator_operator.rs

```diff
@@ -3,65 +3,40 @@
 
 use libra_global_constants::OPERATOR_KEY;
 use libra_management::{
-    constants,
-    error::Error,
-    secure_backend::{SharedBackend, ValidatorBackend},
+    config::ConfigPath, constants, error::Error, secure_backend::SharedBackend,
 };
-use libra_secure_storage::{KVStorage, Value};
+use libra_secure_storage::Value;
 use structopt::StructOpt;
 
 #[derive(Debug, StructOpt)]
 pub struct ValidatorOperator {
+    #[structopt(flatten)]
+    config: ConfigPath,
     #[structopt(long)]
     operator_name: String,
     #[structopt(flatten)]
-    validator_backend: ValidatorBackend,
-    #[structopt(flatten)]
     shared_backend: SharedBackend,
 }
 
 impl ValidatorOperator {
     pub fn execute(self) -> Result<String, Error> {
-        // Get the operator name and verify it exists in the remote storage
-        let operator_name = self.get_and_verify_operator_name()?;
+        let config = self
+            .config
+            .load()?
+            .override_shared_backend(&self.shared_backend.shared_backend)?;
+        let operator_name = self.operator_name;
 
-        // Upload the operator name to shared storage
-        let storage_name = self.shared_backend.name();
-        let mut shared_storage = self
-            .shared_backend
-            .shared_backend
-            .create_storage(storage_name)?;
+        // Verify the operator exists in the shared storage
+        let operator_storage = config.shared_backend_with_namespace(operator_name.clone());
+        let _ = operator_storage.ed25519_key(OPERATOR_KEY)?;
 
-        shared_storage
-            .set(
-                constants::VALIDATOR_OPERATOR,
-                Value::String(operator_name.clone()),
-            )
-            .map_err(|e| {
-                Error::StorageWriteError(storage_name, constants::VALIDATOR_OPERATOR, e.to_string())
-            })?;
+        // Upload the operator name to shared storage
+        let mut shared_storage = config.shared_backend();
+        shared_storage.set(
+            constants::VALIDATOR_OPERATOR,
+            Value::String(operator_name.clone()),
+        )?;
 
         Ok(operator_name)
```

Comment:
We don't need to return the `operator_name` as it was passed by the caller

Reply:
I think the better behavior is to just return () and print success or something, but another PR :D